### ISSUE TITLE
[Bug Report / Compatibility Issue] Paradise Lost and Medieval Weapons with better end compat of the Olvite Longsword/Rapier

### ISSUE THREAD

**Issue by TheShadyyOneYT** [Globox1997/MedievalWeapons#136]
Issue with https://github.com/devs-immortal/Paradise-Lost/issues/805#issuecomment-2297554842

TL;DR: a recipe is making the game crash due to an incompatibility, it seems to be fixed on Paradise Lost's end I think but it would be lovely to have this fixed. Mod is: https://www.curseforge.com/minecraft/mc-mods/paradise-lost

If this is a better end issue just let me know and I will pass it on.

Either this is an issue between Betterend and medieval weapons, betterend x paradise lost, or medieval weapons x paradise lost. to be honest I have no idea. I'm so confused what the issue is, it might be resolved now, but some further insight would be lovely. it was fixed with compatibility with paradise lost, but just something to look into. If it's fixed and its fine, this issue can be closed. I don't think it's anything to serious just I'm very confused what the compatibility issue was to begin with.
